fe/fe-core/src/main/java/com/starrocks/planner/OlapTableSink.java

```diff
@@ -415,6 +415,10 @@ private void setRangeKeys(RangePartitionInfo rangePartitionInfo, Partition parti
                         range.upperEndpoint().getKeys().get(i).treeToThrift().getNodes().get(0));
             }
         }
+
+        if (partition.getName().startsWith(ExpressionRangePartitionInfo.SHADOW_PARTITION_PREFIX)) {
```

Comment:
What's the relationship between SHADOW PARTITION and AUTOMATIC PARTITION?

Reply:
AUTOMATIC PARTITION must have a SHADOW PARTITION, this is to ensure that this table has at least one tablet